CHANGES.md

```diff
@@ -1,6 +1,41 @@
 # Changelog
 
-# 1.4.4 /0.20.4 (libraries)
+# 1.4.5 / 0.20.5 (libraries)
+
+### New
+
+- The eager `AutoMaterializePolicy` now respects data versions, meaning that if a parent is materialized with the same data version as its previous materialization, a second run will no longer be kicked off.
```

Comment:
@Ramshackle-Jamathon this line will need to be removed from the CL

Reply:
done